scripts/deploymentUtils.ts

```diff
@@ -83,78 +84,33 @@ export async function deployContract(
-): Promise<Contract> {
-  console.log("Deploying contract EigenDA x Arbitrum", contractName)
-  const eigenDARollupUtils = await deployContract('EigenDARollupUtils', signer, [], verify)
-  console.log("EigenDARollupUtils deployed at", eigenDARollupUtils.address)
-
-  const eigenda_blob_verifier = await ethers.getContractFactory(contractName, {
-    libraries: {
-        EigenDARollupUtils: eigenDARollupUtils.address
-    }
-  }) as EigenDABlobVerifierL1__factory
-
-  const connectedFactory: ContractFactory = eigenda_blob_verifier.connect(signer)
-
-  let deploymentArgs = [...constructorArgs]
-  if (overrides) {
-    deploymentArgs.push(overrides)
-  }
-
-  const contract: Contract = await connectedFactory.deploy(...deploymentArgs)
-  await contract.deployTransaction.wait()
-  console.log(`New ${contractName} created at address:`, contract.address)
-
-  if (verify)
-    await verifyContract(contractName, contract.address, constructorArgs)
-
-  return contract
-}
-
 // Deploy upgrade executor from imported bytecode
 export async function deployUpgradeExecutor(signer: any): Promise<Contract> {
-  console.log("Deploying contract EigenDA x Arbitrum UpgradeExecutor")
+  console.log('Deploying contract EigenDA x Arbitrum UpgradeExecutor')
   const upgradeExecutorFac = await ethers.getContractFactory(
     UpgradeExecutorABI,
     UpgradeExecutorBytecode
   )
   const connectedFactory: ContractFactory = upgradeExecutorFac.connect(signer)
   const upgradeExecutor = await connectedFactory.deploy()
   await upgradeExecutor.deployTransaction.wait()
-  console.log("Upgrade executor deployed at", upgradeExecutor.address)
+  console.log('Upgrade executor deployed at', upgradeExecutor.address)
   return upgradeExecutor
 }
 
-const L1ServiceManagers = {
-  1: "0x870679E138bCdf293b7Ff14dD44b70FC97e12fc0", // Ethereum
-  5: "0xa3b1689Ab85409B15e07d2ED50A6EA9905074Ee5", // Goerli
-  17000: "0xD4A7E1Bd8015057293f0D0A557088c286942e84b", // Holesky (testnet)
-}
-
 // Function to handle all deployments of core contracts using deployContract function
 export async function deployAllContracts(
   signer: any,
   maxDataSize: BigNumber,
+  eigenDACertVerifierAddress: string,
   verify: boolean = true
 ): Promise<Record<string, Contract>> {
   const isOnArb = await _isRunningOnArbitrum(signer)
-  const chainId = signer.provider.network.chainId
 
-  // If signer is on L1, deploy EigenDABlobVerifierL1 with EigenDAServiceManager as constructor argument
-  // If signer is on L2, deploy EigenDABlobVerifierL2; blob verifier is used as eigenDARollupManager as part of rollup creation
-  const eigenDARollupManager = (L1ServiceManagers.hasOwnProperty(chainId)) ? 
-    await deployBlobVerifierL1('EigenDABlobVerifierL1', signer, [L1ServiceManagers[chainId as keyof typeof L1ServiceManagers]], verify) : await deployContract('EigenDABlobVerifierL2', signer, [], verify)
+  const eigenDACertVerifier = IEigenDACertVerifier__factory.connect(
+    eigenDACertVerifierAddress,
+    signer
+  )
```

Comment:
why arent you deploying a new certverifier instead? think we want each rollup to manage its own certverifier goingh forward

Reply:
was hoping the actual `EigenDACertVerifier` deployment would be a separate script managed in the eigenda monorepo so we wouldn't have manage dual implementations across smart contract dev frameworks (i.e, hardhat, foundry)

cc - @0x0aa0

Reply:
so what's the overall flow for deploying these contracts for a nitro chain? I won't lie I don't really understand whats happening in this PR